marlowe-dashboard-client/src/Template/Lenses.purs

```diff
@@ -74,5 +74,8 @@ _slotParameterDescriptions = prop (SProxy :: SProxy "slotParameterDescriptions")
 _valueParameterDescriptions :: Lens' MetaData (Map String String)
 _valueParameterDescriptions = prop (SProxy :: SProxy "valueParameterDescriptions")
 
-_choiceDescriptions :: Lens' MetaData (Map String String)
-_choiceDescriptions = prop (SProxy :: SProxy "choiceDescriptions")
+_choiceInfo :: Lens' MetaData (Map String ChoiceInfo)
+_choiceInfo = prop (SProxy :: SProxy "choiceInfo")
+
+_choiceDescription :: Lens' ChoiceInfo String
+_choiceDescription = prop (SProxy :: SProxy "choiceDescription")
```

Comment:
I'm just now noticing these lenses (and the other `MetaData` ones) are in `Marlowe.Extended.Metadata`, so we shouldn't be redefining them here. (Did you maybe add these after I put them in the dashboard specific code? Or did I just fail to notice them all along? :man_shrugging:) But I can remove them and use the shared ones in another PR.

Reply:
Yes, just my way of refactoring the code that was already there. But I don't think they are used, actually, I will just remove them